source/develop/release-management/features/network/lldp.html.md

```diff
@@ -0,0 +1,462 @@
+      </property>
+    </properties>
+    <type>6</type>
+  </link_layer_discovery_protocol_element>
+  <link_layer_discovery_protocol_element>
+    <name>System Capabilities</name>
+    <properties>
+      <property>
+        <name>system capabilities</name>
+        <value>Bridge, Router</value>
+      </property>
+      <property>
+        <name>enabled capabilities</name>
+        <value>Bridge, Router</value>
+      </property>
+    </properties>
+    <type>7</type>
+  </link_layer_discovery_protocol_element>
+  <link_layer_discovery_protocol_element>
+    <name>Management Address</name>
+    <properties>
+      <property>
+        <name>management address subtype</name>
+        <value>AFI 802</value>
+      </property>
+      <property>
+        <name>management address</name>
+        <value>00:01:30:f9:ad:a0</value>
+      </property>
+    </properties>
+    <type>7</type>
+  </link_layer_discovery_protocol_element>
+  <link_layer_discovery_protocol_element>
+    <name>Port VLAN Id</name>
+    <oui>32962</oui>
+    <properties>
+      <property>
+        <name>port vlan id</name>
+        <value>488</value>
+      </property>
+    </properties>
+    <subtype>1</subtype>
+    <type>127</type>
+  </link_layer_discovery_protocol_element>
+  <link_layer_discovery_protocol_element>
+    <name>Port VLAN Id</name>
+    <oui>32962</oui>
+    <properties>
+      <property>
+        <name>vlan id</name>
+        <value>488</value>
+      </property>
+      <property>
+        <name>vlan name</name>
+        <value>v2-0488-03-0505</value>
+      </property>
+    </properties>
+    <subtype>3</subtype>
+    <type>127</type>
+  </link_layer_discovery_protocol_element>
+</link_layer_discovery_protocol_elements>
+```
+
+### Graphical User Interface
+
+Graphical User Interface is a [folow-up feature](#follow-up-features)
+
+## Installation/Upgrade
+
+The VDSM package would require the package lldpad and ask the service lldpad to
+start.
+
+Describe how the feature will affect new or existing installations.
+
+## User work-flows
+
+The user gets the information about the networking neighbor via REST-API.
+
+## Dependencies
+
+### oVirt Engine Side
+
+No additional dependencies on oVirt Engine side.
+
+### VDSM Host Side
+
+The package lldpad has to be installed and the demon lldpad has to be running.
+
+## Documentation & External references
+
+[IEEE Std 802.1AB™-2016, IEEE Standard for Local and metropolitan area networks—Station and Media Access Control Connectivity Discovery][1]
+
+[1]: http://standards.ieee.org/getieee802/download/802.1AB-2016.zip
+
+[Linux Ethernet Bonding Driver HOWTO][2]
+
+[2]: https://www.kernel.org/doc/Documentation/networking/bonding.txt
+
+[Ansible Docs » lldp - get details reported by lldp][3]
+
+[3]: http://docs.ansible.com/ansible/lldp_module.html
+
+[OpenStack: Introspection LLDP reporting][4]
+
+[4]: https://specs.openstack.org/openstack/ironic-inspector-specs/specs/lldp-reporting.html
```

Comment:
I would assume we will need the same fields as Ironic does. What are the differences?